packages/fdc3-agent-proxy/src/listeners/DefaultIntentListener.ts

```diff
@@ -24,7 +31,7 @@ export class DefaultIntentListener extends AbstractListener<IntentHandler> {
   }
 
   action(m: IntentEvent): void {
-    this.handleIntentResponse(m);
+    //this.handleIntentResponse(m);
```

Comment:
Should we just delete this line?

Reply:
removed during our meeting earlier (after checking with Rob if it had any relevance)